src/components/FilterMenu.jsx

```diff
@@ -0,0 +1,135 @@
+import {
+  Accordion,
+  AccordionContent,
+  AccordionItem,
+  AccordionTrigger,
+  Flex,
+  Text,
+  Checkbox,
+  RadioGroup,
+  Radio,
+} from "@sparrowengg/twigs-react";
+import {
+  setCategoryFilter,
+  setPriceFilter,
+  setSortedFilter,
+} from "../redux/actions";
+import { useDispatch } from "react-redux";
+const FilterMenu = () => {
+  const dispatch = useDispatch();
+  return (
+    <>
+      <Flex
+        css={{ background: "#ffffff", width: "30%" }}
+        flexDirection="column"
+        justifyContent="center"
+      >
+        <Flex flexDirection="column">
+          <Accordion type="multiple">
+            <AccordionItem value="item-1">
+              <AccordionTrigger>Categories</AccordionTrigger>
+              <AccordionContent>
+                <Flex flexDirection="column" gap="$5">
+                  <Checkbox
+                    id="men's clothing"
+                    size="md"
+                    onChange={() =>
+                      dispatch(setCategoryFilter("men's clothing"))
+                    }
+                  >
+                    <Text>men's clothing</Text>
+                  </Checkbox>
+                  <Checkbox
+                    onChange={() =>
+                      dispatch(setCategoryFilter("women's clothing"))
+                    }
+                    id="women's clothing"
+                    size="md"
+                  >
+                    <Text>women's clothing</Text>
+                  </Checkbox>
+                  <Checkbox
+                    onChange={(e) => dispatch(setCategoryFilter("jewelery"))}
+                    id="jewelery"
+                    size="md"
+                  >
+                    <Text>jewelery</Text>
+                  </Checkbox>
+                  <Checkbox
+                    onChange={(e) => dispatch(setCategoryFilter("electronics"))}
+                    id="electronics"
+                    size="md"
+                  >
+                    <Text>electronics</Text>
+                  </Checkbox>
+                </Flex>
+              </AccordionContent>
+            </AccordionItem>
+
+            <AccordionItem value="item-2">
+              <AccordionTrigger>Price</AccordionTrigger>
+              <AccordionContent>
+                <Flex flexDirection="column" gap="$5">
+                  <Checkbox
+                    id="100rs"
+                    size="md"
+                    onChange={(e) => dispatch(setPriceFilter(100))}
+                  >
+                    <Text>under 100</Text>
+                  </Checkbox>
+                  <Checkbox
+                    onChange={(e) => dispatch(setPriceFilter(200))}
+                    id="200rs"
```

Comment:
@arun-121 this can be just 200 right?, and can you try to move these to a separate constants file,
so what are the contents we need to put in constants : hard coded strings in the code can be moved to constants since the same values can be used in other places of the code.
if you want to change an constant means you don't have to change the entire files which you had used, instead you can change it in the constants itself

Reply:
created a separate folder for constants